presto-docs/src/main/sphinx/security/hive.rst

```diff
@@ -0,0 +1,237 @@
+done using the Simple Authentication and Security Layer (SASL). Kerberos
+authentication for the metastore is configured in the connector's properties
+file using the following properties:
+
+================================================== ============================================================ ==========
+Property Name                                      Description                                                  Default
+================================================== ============================================================ ==========
+``hive.metastore.authentication.type``             One of ``SIMPLE`` or ``SASL``. When set to ``SASL`` the Hive ``SIMPLE``
+                                                   connector will connect to the Hive metastore Thrift service
+                                                   using SASL and authenticate with Kerberos. When using the
+                                                   default value of ``SIMPLE``, Kerberos authentication is
+                                                   disabled and no other properties need to be configured.
+
+``hive.metastore.principal``                       The Kerberos principal of the Hive metastore. The Presto     
+                                                   coordinator will use this to authenticate the Hive
+                                                   metastore.
+
+``hive.metastore.presto.principal``                The Kerberos principal that Presto will use when connecting
+                                                   to the Hive metastore.
+
+``hive.metastore.presto.keytab``                   The path to the keytab file that contains a key for the
+                                                   principal specified by hive.metastore.presto.principal.
+
+================================================== ============================================================ ==========
+
+``SIMPLE`` authentication
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: none
+
+    hive.metastore.authentication.type=SIMPLE
+
+``SIMPLE`` authentication is the default authentication mechanism for the Hive
+metastore. When using ``SIMPLE`` authentication, Presto connects to an
+unsecured Hive metastore. Kerberos is not used.
+
+``SASL`` authentication
+^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: none
+
+    hive.metastore.authentication.type=SASL
+    hive.metastore.principal=hive/_HOST@EXAMPLE.COM
+    hive.metastore.presto.principal=presto@EXAMPLE.COM
+    hive.metastore.presto.keytab=/etc/presto/hive.keytab
+    
+When SASL is enabled for the Hive metastore Thrift service, Presto will connect
+as the principal specified by the property ``hive.metastore.presto.principal``.
+Presto will authenticate using the keytab specified by
+``hive.metastore.presto.keytab``, and will verify that the identity of the
+metastore matches ``hive.metastore.principal``. The special value _HOST is
```

Comment:
I dropped the _HOST part out since the hostname of the local host will be substituted. This is the host the hive metastore is running on for at most one host in a Presto cluster :-)